testutils/build.gradle

```diff
@@ -56,11 +64,86 @@ android {
 
 }
 
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+}
+
+tasks.whenTaskAdded { task ->
+    if (task.name == 'assembleDebug' || task.name == 'assembleRelease') {
+        task.dependsOn 'sourcesJar'
+    }
+}
+
+project.afterEvaluate{
+    publishing {
+        publications {
+            aar(MavenPublication) {
+                artifact sourcesJar
+                groupId 'com.microsoft.identity'
+                artifactId = 'testutils'
+                version = getAppVersionName()
+
+                pom.withXml {
+
+                    // Dependencies
+                    def dependenciesNode = asNode().appendNode('dependencies')
+
+                    //Iterate over the implementation dependencies (we don't want the test ones), adding a <dependency> node for each
+                    configurations.implementation.allDependencies.each {
+                        if (it.group != null && it.name != null) {
+                            def dependencyNode = dependenciesNode.appendNode('dependency')
+                            dependencyNode.appendNode('groupId', it.group)
+                            dependencyNode.appendNode('artifactId', it.name)
+                            dependencyNode.appendNode('version', it.version)
+                        }
+                    }
+                }
+            }
+        }
+
+        repositories {
+            maven {
+                name "vsts-maven-adal-android"
+                url "https://identitydivision.pkgs.visualstudio.com/_packaging/AndroidADAL/maven/v1"
+                credentials {
+                    username project.ext.vstsUsername
+                    password project.ext.vstsPassword
+                }
+            }
+        }
+    }
+}
+
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation project(':common')
+
     implementation project(':keyvault')
     implementation project(':labapi')
+
+    implementation(group: 'com.microsoft.identity', name: 'common', version: '0.0.22')
```

Comment:
What version of common is this? Looks very old

Reply:
Good catch.  I'm going to point this to 0.0.+.  Using 0.0.<patch> to ensure that no one tries to ship one of these.

Reply:
but why are we not using the latest? (`3.0.4`)

Reply:
I thought 0.0 made it clear that this isn't intended to be used... in any production scenario.